.github/workflows/ci-contrib.yml

```diff
@@ -24,6 +24,7 @@ jobs:
       fail-fast: false
       matrix:
         gem:
+          - semconv
```

Comment:
I'm a little worried about the name, but am not quite sure what to call it instead. Mostly leaving this as a note to think about later.